### PR TITLE
Release v0.1.5rc1

### PR DESCRIPTION
**Pull request by bouthilx**
### New commands 
New commands `list`, `status` and `info` now enable simple monitoring and inspection of experiments. The command `list` prints out all the experiments' name so you can easily keep track of what is available in your database. The command `status` gives a summary of experiments' status, with number of trials in different status (`completed`, `reserved`, etc) and best objective found so far. Finally the command `info` prints out all details of a given experiment, command line used, search space, algorithm configuration and current statistics.

### Trial interruption

So far interrupted trials would keep the status `reserved` and wouldn't be available for selection   
again, so they would be lost somehow. They will now be set to `interrupted` and will be available
for resumption. If user's script does not support checkpointing internally, the process will simply be re-executed from scratch.

## New features
- New command `list` (#208, #216 )
- New command `status` (#222, #229)
- New command `info` (#211)
- New StorageProtocol (#221)
- Handling of interrupted trials (#197, #202, #203, #235) 
- Limit number of broken trials before experiment stop execution (#204)
- Add working_dir attribute to Trial (#205)
## Breaking Changes
- Deprecate pack/unpack in orion.algo.space (#236)
## Bug Fixes
- Fix code conflict detection when not using code versioning (#207)
- Add $ne and handle non-supported operations in EphemeralDB (#214)
- Fix ScriptConfigConflict for data with non-string values (#215)
- Fix race conditions during algo updates (#217)
- Fix experiment views (#218, #223, #234)
- Make sure higher shape points are not `ndarray` objects (#224, #236)
## Other Improvements
- Mute (intense) logging of filelock (#201)
- Functional tests for user script config (#212)
- Use yaml.safe_load instead of yaml (#219)
- Remove upsert from `write` (#220)
## Documentation Improvements
- Add documentation for new commands `list`, `info` and `status` (#226, #228)
- Add documentation on Search Space (#227)

### CODE REVIEW

File: src/orion/algo/space.py [hunk 1]
```diff
@@ -818,52 +818,3 @@ def __repr__(self):
         """Represent as a string the space and the dimensions it contains."""
         dims = list(self.values())
         return "Space([{}])".format(',\n       '.join(map(str, dims)))
-
-
-def pack_point(point, space):
```
Comment (DatCorno) [inline src/orion/algo/space.py]: Wasn't this supposed to be left in?

Comment (bouthilx) [reply src/orion/algo/space.py]: Indeed! Good catch!

**Comment by bouthilx** [comment]
[Release v0.1.5](https://trello.com/c/03AGd5lt/182-release-v015)

**Comment by bouthilx** [comment]
[Release v0.1.5](https://trello.com/c/03AGd5lt/182-release-v015)

**Pull request merged**